test/manual/end-test.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+var test = require('tape')
+var leveljs = require('../..')
+var testCommon = require('../testCommon')
+var db
+
+test('setUp', testCommon.setUp)
+
+test('create', function (t) {
+  db = leveljs(testCommon.location())
+  db.open(function (err) {
+    t.ifError(err)
+
+    db.put('key', 'value', function (err) {
+      t.ifError(err)
+      t.end()
+    })
+  })
+})
+
+test('repeated end', function (t) {
+  function loop (n) {
+    if (n > 5e3) return t.end()
+
+    var iterator = db.iterator()
+
+    iterator.next(function (err, key, value) {
+      if (err) throw err
```

Comment:
I presume these throw statements is to not spam the tests too much?

Reply:
Yeah, in an earlier iteration I got thousands of errors ;)